assets/src/components/formattedHeader/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Formatted header for titling screens/sections.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+class FormattedHeader extends Component {
+	/**
+	 * Render.
+	 */
+	render() {
+		const { headerText, subHeaderText } = this.props;
+		const classes = 'newspack-formatted-header' + ( !! subHeaderText ? ' has-subheader' : '' );
```

Comment:
You could use the [`classnames`](https://jedwatson.github.io/classnames/) utility for a (maybe?) more elegant approach here. In this case your approach is fewer characters and requires no dependency so it's superior, but just mentioning this for future cases where we get into complex className logic. It would look like this:

`import classnames from 'classnames';`

then...

```
const classes = classnames(
			'newspack-formatted-header',
			!! subHeaderText ? 'has-subheader' : null
		);
```

Reply:
Yeah, I'm sure that utility will come in handy a lot, so let's just start using it. 👍